src/django_htmx/management/commands/downloadhtmx.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import annotations
+
+import os
+from pathlib import Path
+from typing import TYPE_CHECKING
+from typing import Any
+from urllib.parse import urlparse
+from urllib.parse import urlunparse
+from urllib.request import urlopen
+
+from django.conf import settings
+from django.core.checks import Tags
+from django.core.management.base import BaseCommand
+from django.core.management.base import CommandError
+
+if TYPE_CHECKING:
+    from argparse import ArgumentParser
+
+
+class Command(BaseCommand):
+    help = "Download the given htmx version and the extensions."
+    requires_system_checks = [Tags.staticfiles]
+
+    def add_arguments(self, parser: ArgumentParser) -> None:
+        parser.add_argument(
+            "version",
+            nargs="?",
+            help="version to download, e.g. '1.0.1. default: latest",
+        )
+        parser.add_argument(
+            "--dest",
+            "-d",
+            type=Path,
+            default=None,
+            help="where to download HTMX to. default: the first STATICFILES_DIR",
+        )
+
+    def handle(self, *args: str, **options: Any) -> None:
+        dest_dir = options["dest"]
+        if dest_dir is None:
+            if len(settings.STATICFILES_DIRS) == 0:
+                raise CommandError(
+                    "Destination location not defined. "
+                    "Use '--dest' or set STATICFILES_DIRS in your settings"
+                )
+            dest_dir = Path(settings.STATICFILES_DIRS[0])
+
+        package_spec = "htmx.org"
+        if options["version"] is not None:
+            package_spec += f"@{options['version']}"
+
+        actual_version = options["version"]
+        for file in ["htmx.js", "ext/debug.js", "ext/event-header.js"]:
```

Comment:
Extensions should be selected by the user on the command line, not hardcoded. The previous script hardcoded these extensions as they’re used in the example app. Add a `--ext` option.

Reply:
done in bd4cd29822063198ae59f159bfa885c5a40a45ef